local-volume/bootstrapper/deployment/kubernetes/bootstrapper.yaml

```diff
@@ -1,21 +1,21 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: local-volume-bootstrapper
+  name: local-volume-provisioner-bootstrap
   namespace: kube-system
   labels:
-    app: local-volume-bootstrapper
+    app: local-volume-provisioner-bootstrap
 spec:
   restartPolicy: Never
   serviceAccountName: local-storage-bootstrapper
   containers:
   - name: bootstrapper
-    image: local-volume-bootstrapper:dev
+    image: quay.io/external_storage/local-volume-provisioner-bootstrap
```

Comment:
I don't think I have permission to push the image, can you push one once it's ready?